src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -540,6 +540,57 @@
   <data name="Event_Hookup_Command_Handler" xml:space="preserve">
     <value>Generate Event Subscription Command Handler</value>
   </data>
+  <data name="Remove_unnecessary_usings" xml:space="preserve">
+    <value>Remove unnecessary usings</value>
+  </data>
+  <data name="Sort_usings" xml:space="preserve">
+    <value>Sort usings</value>
+  </data>
+  <data name="Format_document_settings" xml:space="preserve">
+    <value>Format Document Settings (Experiment) </value>
+  </data>
+  <data name="All_csharp_formatting_rules" xml:space="preserve">
+    <value>All C# Formatting Rules (indentation, wrapping and spacing)</value>
+  </data>
+  <data name="Fix_accessibility_modifiers" xml:space="preserve">
+    <value>Fix accessibility modifiers</value>
```

Comment:
What does it mean to "fix" these?

Reply:
➡️ We'll keep this comment in mind for our strings pass at the end.